src/reducers/breakpoints.js

```diff
@@ -13,7 +13,7 @@ const { updateObj } = require("../utils/utils");
 const I = require("immutable");
 const makeRecord = require("../utils/makeRecord");
 
-import type { Breakpoint, Location } from "devtools-client-adapters";
+import type { Breakpoint, Location } from "devtools-client-adapters/src/types";
```

Comment:
Was thinking that too. 

Also it might be possible to do what react dies and set up a declaration so that we don't import at all...